tools/ci_build/github/linux/docker/scripts/install_openvino.sh

```diff
@@ -7,16 +7,16 @@ o) OPENVINO_VERSION=${OPTARG};;
 esac
 done
 
-OPENVINO_VERSION=${OPENVINO_VERSION:=2019_R1.1}
-git clone https://github.com/opencv/dldt.git /data/dldt/openvino_2019.1.144
+OPENVINO_VERSION=${OPENVINO_VERSION:=2019_R3.1}
+git clone https://github.com/opencv/dldt.git /data/dldt/openvino_2019.3.376
 
-export INTEL_CVSDK_DIR=/data/dldt/openvino_2019.1.144
+export INTEL_CVSDK_DIR=/data/dldt/openvino_2019.3.376
 apt-get update && apt-get -y  install libusb-1.0-0-dev
 
 cd ${INTEL_CVSDK_DIR}/inference-engine
+git checkout tags/$OPENVINO_VERSION -b $OPENVINO_VERSION
```

Comment:
if fetching/building openvino from source isn't supported anymore, can remove all these lines related to git

Reply:
we are doing the openvino from source for CI for CPU